tools/build-release-branch.sh

```diff
@@ -154,11 +158,11 @@ TMP_REMOTE_BUILT_VERSION="/tmp/jetpack"
 TMP_LOCAL_BUILT_VERSION="/tmp/jetpack2"
 
 # Make sure we don't have uncommitted changes.
-if [[ -n $( git status -s --porcelain ) ]]; then
-	echo "Uncommitted changes found."
-	echo "Please deal with them and try again clean."
-	exit 1
-fi
+# if [[ -n $( git status -s --porcelain ) ]]; then
+# 	echo "Uncommitted changes found."
+# 	echo "Please deal with them and try again clean."
+# 	exit 1
+# fi
 
 # Check the command
 if [[ 'new' == $COMMAND || '-n' == $COMMAND ]]; then
```

Comment:
What do you think about using `./tools/version-update.sh -v` to update the version when someone updates a release branch with `tools/build-release-branch.sh -u`? I think it would be useful when you want to release a new Beta during a Beta period, for example.

It would require an additional prompt in there so one can specify that they want to update to `x.x-beta2` for example, but I think it would be useful.

Reply:
Done!